src/pages/developing/keyboard.md

```diff
@@ -7,7 +7,55 @@ previousUrl: '/docs/developing/hardware-back-button'
 
 Customizing and accounting for the presence of an on-screen keyboard are two common roadblocks developers face when building mobile apps and PWAs. This guide will provide an introduction to the various tools available for managing the on-screen keyboard in your application.
 
-## inputmode
+## Working with Forms
+
+Keyboards and forms in single page applications present a unique challenge given how each platform tries to automatically resize webviews or scroll inputs into view. The biggest challenge is ensuring that an input is not covered by the keyboard when it is focused. Ionic Framework provides a few utilities that attempt to bridge the gap between the browser's default behavior and what is typically expected by users in mobile applications.
+
+### iOS WebView Resizing
+
+In Capacitor and Cordova applications, the default iOS webview behavior is to resize the webview when the on-screen keyboard is presented. When the keyboard is dismissed, the webview will then resize back to its full height. As a result, you may notice a flash of black or white where the keyboard used to be. This is empty space, and the webview will quickly resize to take up this space. 
+
+This flash may be undesirable to some developers. To avoid this, developers can change the resize mode for the Keyboard plugin in either Capacitor or Cordova. For example, developers may wish to disable all webview resizing by modifying their `capacitor.config.json` file:
+
+```json
+{
+  "plugins": {
+    "Keyboard": {
+      "resize": "none"
+    }
+  }
+}
+```
+
+> See [Capacitor Keyboard Configuration](https://capacitorjs.com/docs/apis/keyboard#keyboard-configuration-ios-only) for a full list of options. 
+
+There are still a few issues that exist when it comes to ensuring an input is not covered by the keyboard. This is where the [Scroll Assist](#scroll-assist) and [Scroll Padding](#scroll-padding) utilities can help.
+
+### Scroll Assist
```

Comment:
to clarify, these two Scroll sections are basically explaining that things might still be janky on PWAs, and not much we can do about it. yea?

Reply:
More or less. There _are_ a few things that I am doing for v5.5.1 that may knock one or two items off this list.

Reply:
Well this section describes things that may be janky event with scroll assist: https://github.com/ionic-team/ionic-docs/pull/1767/files#diff-79bce33e85b3dcef172d7f21c3f1926fcfce166cd59a53349b097ad4fbc67c8cR40-R44

But both scroll sections are intended to clarify how Ionic Framework works around other issues.